modules/party/mod_settings/db.xml

```diff
@@ -98,7 +98,7 @@
 				<type>datetime</type>
 				<null></null>
 				<key></key>
-				<default></default>
+				<default>0000-00-00 00:00:00</default>
```

Comment:
What makes this datetime field different from the others when it comes to the default value?